R/Raylib/build_tarballs.jl

```diff
@@ -23,13 +23,23 @@ if [[ "${target}" == *-freebsd* ]]; then
     CFLAGS="${CFLAGS} -D__BSD_VISIBLE"
 fi
 CFLAGS="${CFLAGS} -DSUPPORT_EVENTS_AUTOMATION -DSUPPORT_FILEFORMAT_BMP -DSUPPORT_FILEFORMAT_JPG"
-make -j${nproc} USE_EXTERNAL_GLFW=TRUE PLATFORM=PLATFORM_DESKTOP RAYLIB_LIBTYPE=SHARED RAYLIB_MODULE_RAYGUI=TRUE RAYLIB_MODULE_PHYSAC=TRUE
+
+if [[ "${target}" == *-mingw* ]]; then
+    # raylib.rc.data is broken for x64, remove it from compilation
+    sed -i 's+$(RAYLIB_RES_FILE)+ +g' Makefile
+    # we need to specify the OS in the flags to make for Windows
+    make -j${nproc} USE_EXTERNAL_GFLW=TRUE PLATFORM=PLATFORM_DESKTOP RAYLIB_LIBTYPE=SHARED RAYLIB_MODULE_RAYGUI=TRUE RAYLIB_MODULE_PHYSAC=TRUE OS=Windows_NT 
+    # the .dll produced is called raylib.dll not libraylib.dll, products looks for "libraylib"
```

Comment:
The first argument of [`LibraryProduct`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct) can be a vector with the different basenames. There is no need to rename the file